awscrt/mqtt.py

```diff
@@ -21,10 +21,29 @@
 
 
 class QoS(IntEnum):
-    """Quality of Service"""
+    """Quality of Service enumeration"""
+
     AT_MOST_ONCE = 0
+    """QoS 0 - At most once delivery
```

Comment:
Should we cite the spec sections like we do in native?

Reply:
good idea